spec/acceptance/sequence_class_methods_spec.rb

```diff
@@ -0,0 +1,112 @@
+describe "Sequence Class Method" do
```

Comment:
(complementing another comment)

Looks like those specs could be moved to the existing `spec/factory_bot/sequence_spec.rb`